spec/rubocop/cop/rspec/capybara/current_path_expectation_spec.rb

```diff
@@ -0,0 +1,29 @@
+RSpec.describe RuboCop::Cop::RSpec::Capybara::CurrentPathExpectation do
+  subject(:cop) { described_class.new }
+
+  it 'flags violations for `expect(current_path)`' do
+    expect_offense(<<-RUBY)
+      expect(current_path).to eq("/callback")
+      ^^^^^^ Do not set an RSpec expectation on `current_path` in Capybara feature specs - instead, use the `have_current_path` matcher on `page`
+    RUBY
+  end
+
+  it 'flags violations for `expect(page.current_path)`' do
+    expect_offense(<<-RUBY)
+      expect(page.current_path).to eq("/callback")
+      ^^^^^^ Do not set an RSpec expectation on `current_path` in Capybara feature specs - instead, use the `have_current_path` matcher on `page`
+    RUBY
+  end
+
+  it 'doesn\'t flag a violation for other expectations' do
+    expect_no_offenses(<<-RUBY)
+      expect(current_user).to eq(user)
+    RUBY
+  end
+
+  it 'doesn\'t flag a violation for other references to `current_path`' do
```

Comment:
Nitpick: If you use double quotes around the string, you don’t need to escape the `'` in the middle. (also in line 18).